examples/pagerank/pagerank.cc

```diff
@@ -0,0 +1,116 @@
+/** Copyright 2022 Alibaba Group Holding Limited.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+#include <filesystem>
+#include <iostream>
+
+#include "arrow/api.h"
+#include "arrow/filesystem/api.h"
+
+#include "gar/graph.h"
+#include "gar/graph_info.h"
+#include "gar/reader/arrow_chunk_reader.h"
+#include "gar/writer/arrow_chunk_writer.h"
+
+
+int main(int argc, char* argv[]) {
+  // read file and construct graph info
+  std::string path = "/tmp/ldbc_sample/parquet/ldbc_sample.graph.yml";
```

Comment:
This is a little hard-code. Use the relative path to `test/gar-test` or add a soft link to it maybe better

Reply:
My purpose is to make this example a new project over the GAR library.